plow/backends/backend.flow

```diff
@@ -1,12 +1,27 @@
 import plow/backends/back_ast;
 
 export {
+
+	// The general info about a backend.
+	BackendSpec(
+		keywords : Set<string>, // A set of target language keywords
+		natives : Set<string>,  // A set of all supported natives
+		onError : (int, string) -> void,
```

Comment:
The pos thing does not make sense here, since that is only relevant if you have the source file.

Reply:
So either add the id it is related to as well, or do a look up for the file as an argument.

Reply:
onError : (int, string) -> void  in Backend changed to just onError: (string) -> void, line and column are processed in a separate function